packages/app/pages/organization/user/[uid]/edit.tsx

```diff
@@ -42,6 +42,9 @@ export default function Edit(): ReactElement {
   ) {
     return <DefaultErrorPage statusCode={404} />
   }
+  if (!uid) {
```

Comment:
This wouldn't necessarily work. Page will return 400 if data is still loading


You should add a check here:
```
return profile &&
    profile.organization.organizationRole == OrganizationRole.ADMIN &&
    organization && uid ?
```